Git.md

```diff
@@ -0,0 +1,82 @@
+```
+
+## Flow
+
+We use the [Git Flow](https://nvie.com/files/Git-branching-model.pdf) workflow model.
+
+### Master
+The `master` branch is one of the two branches with an infinite lifetime. The source code of `HEAD` always reflects a production-ready state.
+
+### Develop
+
+`develop` is the second branch with an infinite lifetime. The `develop` branch source code of `HEAD` always reflects a state with the latest delivered development changes for the next releases.
+
+When the source code in `develop` branch reaches a stable point and is ready to be released, all of the changes should be merged back into `master` and tagged with a release number.
+
+### Feature
+
+Next to the main `master` and `develop` branches the Flow model uses a variety of supporting branches to aid parallel development between team members, ease tracking of features, prepare for production releases, and to assist in quickly fixing live production problems.
+Unlike the two main branches, these branches have a limited lifetime.
+
+Feature branches may branch of from `develop` and must be merged back into `develop`. Feature branch naming convention is as follows: `feature/*`, for example `feature/login-page`
+
+Feature branches are used to develop new features for the upcoming or a distant future release. These branches exist as long as the feature is in development. Eventually, they are merged back into `develop` or are discarded.
+
+
+### Hotfix
+
+Hotfix branches are created if you need to immediately act upon an undesired state of a live production version. If a critical bug in production must be resolved immediately, a hotfix branch can be branched off from `master` branch. Similar to the feature branches, the hotfix branches are named `hotfix/*`. A hotfix branch should be merged back into `master` and `develop`.
+
+
+## Committing
+
+A commit should round up only the related changes. For example, fixing two different bugs should produce two separate commits. Small, atomic commits make it easier for other developers to understand the changes and roll them back if needed.
+
+Committing often will keep your commits small and help you commit only related changes. It will also allow you to share your code more frequently with others. Furthermore, it will help with avoiding merge conflicts.
+
+Don't use `git add -A` to stage changes. If you have a lot of changes you can filter through them with `git add -p` and stage only relevant changes.
+
+
+### Commit messages
+
+Writing good commit messages is important. A clear commit log will be easier to navigate through and make it easier for other people to follow along. You can read on how to write proper [here](https://chris.beams.io/posts/git-commit/).
+
+
+## Pull requests
+
+Avoid creating pull requests that contain a lot of changes. If you are implementing a feature that requires a lot of file changes, try breaking it up into multiple smaller features which results in multiple pull requests.
```

Comment:
Maybe link the test "a lot of changes" to https://www.urbandictionary.com/define.php?term=KPR ? 😄